circuits/lib/query/query.circom

```diff
@@ -1,18 +1,25 @@
 pragma circom 2.1.1;
-include "../../../node_modules/circomlib/circuits/mux3.circom";
+include "../../../node_modules/circomlib/circuits/mux1.circom";
+include "../../../node_modules/circomlib/circuits/mux4.circom";
 include "../../../node_modules/circomlib/circuits/bitify.circom";
 include "../../../node_modules/circomlib/circuits/comparators.circom";
+include "../../../node_modules/circomlib/circuits/gates.circom";
 include "comparators.circom";
 
 /*
   Operators:
- "0" - noop, skip execution. Ignores all `in` and `value` passed to query, out 1
- "1" - equals
- "2" - less-than
- "3" - greater-than
- "4" - in
- "5" - notin
- "6" - not equals
+    0 - noop, skip execution. Ignores all `in` and `value` passed to query, out 1
+    1 - equals
+    2 - less than
+    3 - greater than
+    4 - in
+    5 - not in
+    6 - not equals
+    7 - less than or equal
+    8 - greater than or equal
+    9 - between
+    16 - selective disclosure (16 = 10000 binary)
+    17 - nullify (17 = 10001 binary)
```

Comment:
Generally, the idea to mix query operations and modifier operations in one template/inputs looks strange as opposed to splitting it to different templates/inputs. However, I may not see other reasons in favour the decision, like gas consumption, etc.

Reply:
yeah, but mux outside and checks for only lower 4 bits of operator inside Query template were looking worse. Maybe you can propose something better.